datarobot_provider/example_dags/datarobot_create_project_from_ai_catalog_dag.py

```diff
@@ -0,0 +1,52 @@
+# Copyright 2023 DataRobot, Inc. and its affiliates.
+#
+# All rights reserved.
+#
+# This is proprietary source code of DataRobot, Inc. and its affiliates.
+#
+# Released under the terms of DataRobot Tool and Utility Agreement.
+"""
+Example of DAG to upload local file to DataRobot AICatalog as a static dataset,
+then using this stored dataset as a training data to create a DataRobot Project.
+
+Config example for this dag:
+{
+    "dataset_file_path": "/path/to/local/file",
+    "project_name": "test project",
+    "unsupervised_mode": False,
+    "use_feature_discovery": False
+}
+"""
+from datetime import datetime
+
+from airflow.decorators import dag
+
+from datarobot_provider.operators.ai_catalog import UploadDatasetOperator
+from datarobot_provider.operators.datarobot import CreateProjectOperator
+
+
+@dag(
+    schedule_interval=None,
+    start_date=datetime(2022, 1, 1),
```

Comment:
What is this value for -- should we generate it dynamically?

Reply:
this value will be changed by customer when they will reuse this example, so its more like placeholder for DAG execution start date